### PR TITLE
Add date range filtering for Records

### PR DESCRIPTION
**Pull request by moradology**

### CODE REVIEW

File: ashlar/filters.py [hunk 1]
```diff
@@ -84,6 +87,66 @@ class Meta:
...
+        """Filter records by date
+
+        Arguments
+        :param request: django rest framework request instance
+        :param queryset: queryset to apply filter to
+        :param view: view that this filter is being used by
+
+        QUERY PARAMS
+        :param valid_from: ISO 8601 timestamp
+        :param valid_to: ISO 8601 timestamp
```
Comment (ddohler) [inline ashlar/filters.py:120]: Looks like these and the big comment above need to be updated to match this functionality.

File: ashlar/migrations/0014_auto_20150929_1551.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    dependencies = [
+        ('ashlar', '0013_auto_20150514_1712'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='record',
+            options={'ordering': ('-created',)},
+        ),
+        migrations.AlterField(
```
Comment (ddohler) [inline ashlar/migrations/0014_auto_20150929_1551.py]: What are these two `AlterField`s doing?

**Comment by ddohler** [comment]
+1 after addressing those two items.

File: ashlar/migrations/0014_auto_20150929_1551.py (hunk 2 above)
Comment (moradology) [inline ashlar/migrations/0014_auto_20150929_1551.py]: Good question - autogenerated by migrations. Do you have any thoughts on it?

Comment (ddohler) [inline ashlar/migrations/0014_auto_20150929_1551.py]: I remember that we changed the srid a few times several weeks ago but that's it -- does it match what's in the models file? @kshepard , any thoughts?

Comment (moradology) [inline ashlar/migrations/0014_auto_20150929_1551.py]: Yeah, I'm using the new data without a problem. tests all pass

Comment (ddohler) [inline ashlar/migrations/0014_auto_20150929_1551.py]: Hmm, if you check out `develop` and generate migrations, does it still generate this? It might have been a change from a while ago that didn't get a migration when it should have. If so, it's probably safe to include it but I'd split it out into a separate commit.

Comment (kshepard) [inline ashlar/migrations/0014_auto_20150929_1551.py]: The srid was changed to 4326 in #45. In the PR, it looks like a previous migration was changed to reflect this update rather than a new migration added. I wonder if your DB was still using the old srid, because you ran migrations with the old version of that migration before it was updated. I suspect if you destroyed/recreated your DB VM and then ran makemigrations, these calls to AlterField wouldn't be generated.
